webapp/app/dataExport/components/columnSelect/columnSelect.js

```diff
@@ -25,17 +25,17 @@ const ColumnSelect = (props) => {
 
       <div className="export__form-section-variables">
         {columns.map((column) => {
-          const selected = selectionColumns.includes(column)
+          const selected = !!selectionColumns.find(({ param }) => param === column)
```

Comment:
Would it be better to add the column to the selection and not the custom object {param, label} ?
Then you can have a Column file with method getLabel